pull_data.R

```diff
@@ -0,0 +1,33 @@
+library(sbtools)
+library(data.table)
+library(readxl)
+
+source("auth.R")
```

Comment:
I added "auth.R" to the gitignore file, so hopefully no one would accidentally check it in, but then yeah, it would be a file that just has the function `authenticate_sb`. Once the data is public, we can delete this line/file. Sound OK? An alternative would be to have the `authenticate_sb` in this script without the username/password. It's just a bit more annoying because when you run it, you have to manually enter your credentials. Easy enough to change though if you think that would be better.